crypto/ed25519.go

```diff
@@ -75,7 +75,10 @@ func GeneratePrivateKey() (PrivateKey, error) {
 // PublicKey returns a PublicKey associated with the Ed25519 PrivateKey p.
 // The PublicKey is the last 32 bytes of p.
 func (p PrivateKey) PublicKey() PublicKey {
-	return PublicKey(p[PublicKeyLen:])
+	rpk := p[PrivateKeyLen:] // privateKey == private|public
```

Comment:
You can put a TODO here for #23 so we make sure to catch it.

Reply:
Or just leave it. LGTM now.

Reply:
Added the `PrivateKeyLen` variable to be more explicit, this MR is just some minor clean-ups.